src/server/tran_server.cpp

```diff
@@ -420,6 +414,8 @@ int
 tran_server::reset_main_connection ()
 {
   auto &conn_vec = m_page_server_conn_vec;
```

Comment:
Why is `conn_vec` still needed?
And why is it outside the lock?

Reply:
It's just making a reference. I think it is safe without the lock, isn't it?

I think I can remove it, by the way.